File: Files/task5.py
#
#  task5.py
#
#  Created by Angelos Stavrou
#  Python program to explain os.urandom() method
        
# importing os module
import os
      
# Declaring size
size = 16 # This is in bytes
  
# Using os.urandom() method
result = os.urandom(size)
      
# Print the random bytes string
# Output will be different everytime
print(result.hex())

File: Files/task5.c
//
//  task5.c
//
//
//  Created by Angelos Stavrou
//

#include <stdio.h>
#include <stdlib.h>
int main()
{
    #define LEN 16    // 128 bits

    unsigned char *key = (unsigned char *) malloc(sizeof(unsigned char)*LEN);
    FILE* random = fopen("/dev/urandom", "r");
    fread(key, sizeof(unsigned char)*LEN, 1, random);
    fclose(random);
    return 0;
}
